docs/docs/middlewares/tracing.md

```diff
@@ -53,7 +53,7 @@ func main() {
 func processUserLoggedInEvent(ctx context.Context, message UserLoggedInEvent) error {
 	// here, you can focus on your business logic.
 	fmt.Printf("processing %v\n", message)
-	time.Sleep(time.Millisecond * 200) // simulate long running process
+	time.Sleep(time.Millisecond * 200) // simulate long-running process
```

Comment:
Is it possible to make it constant? And use it?

Reply:
this is a markdown document for documentation, I think I'm gonna keep it like this though :)

Reply:
Right, may be on top of the Markdown file we can define it. So that someone who is reviewing/reading this markdown can understand. 
But its fine I think :)